cmd/gateway-conformance/main.go

```diff
@@ -190,6 +190,12 @@ func main() {
 						if err != nil {
 							return err
 						}
+
+						// TODO: merge IPNS records somehow
+						err = copyFiles(fxs.IPNSRecords, directory)
```

Comment:
:thought_balloon: Some related thoughts/ideas in https://github.com/ipfs/specs/issues/369.
(tldr fixtures could be included as blocks in a CAR with immutable data, as blocks with `ipns-record` codec)